src/main/java/reactor/netty/udp/UdpServer.java

```diff
@@ -136,7 +136,9 @@ public final UdpServer bindAddress(Supplier<? extends SocketAddress> bindAddress
 	 * enriched bootstrap.
 	 *
 	 * @return a new {@link UdpServer}
+	 * @deprecated  as of 0.9.10. Use the methods exposed on {@link UdpServer} level.
```

Comment:
same

Reply:
This method is removed in master, so I'll add 1.0.0

Reply:
done